package.json

```diff
@@ -57,11 +54,11 @@
     "broccoli-asset-rev": "^3.0.0",
     "ember-cli": "~3.28.0",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-deprecation-workflow": "^1.0.1",
+    "ember-cli-deprecation-workflow": "^3.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~3.24",
+    "ember-data": "~3.28.13",
```

Comment:
Needed to avoid use of Ember global - later ember versions remove the ember global